sql/src/main/java/org/apache/druid/sql/calcite/planner/DruidTypeSystem.java

```diff
@@ -29,7 +29,8 @@ public class DruidTypeSystem implements RelDataTypeSystem
   public static final DruidTypeSystem INSTANCE = new DruidTypeSystem();
 
   /**
-   * Druid uses millisecond precision for timestamps internally. This is also the default at the SQL layer.
+   * Druid uses millisecond precision for timestamps internally. This is also
+   * the default at the SQL layer.
```

Comment:
Interesting. While the code can sprawl to 120 characters, it is actually quite hard to read comments at that width. Books, magazines and newspapers have limited width for this reason. The old 120-column green-bar paper has long since given way to 80 column (approx.) letter-sized pages.

I wonder, should we allow our code to be 120 characters, but encourage 80 for comments? Here's [a random reference](https://spin.atomicobject.com/2016/09/24/improving-web-typography/#:~:text=Takeaway%3A%20Set%20text%20column%20widths,around%2075%20characters%20helps%20readability.) which suggests I'm not alone in this opinion.

If we want to compress the code, loosening our rules for `{` (put on same line as method declaration) and ')' (append to the last parameter) is a better way to do so.

In any case, reverted the change here.

Reply:
> I wonder, should we allow our code to be 120 characters, but encourage 80 for comments? Here's [a random reference](https://spin.atomicobject.com/2016/09/24/improving-web-typography/#:~:text=Takeaway%3A%20Set%20text%20column%20widths,around%2075%20characters%20helps%20readability.) which suggests I'm not alone in this opinion.

Yeah I guess the issue is that shorter lines lead to "taller" code, and displays are generally wider than they are tall.

My feeling is that we need not be authoritarian: the only standard we need across the codebase is that line length for new code should be somewhere between 80 and 120 chars. I wouldn't have said anything here if this was a new comment. I only mentioned it because I didn't think there was a reason to change the pre-existing formatting.